src/arch/mod.rs

```diff
@@ -309,7 +309,13 @@ pub mod mie {
     /// Constant to filter out SIE bits of mstatus
     pub const SIE_FILTER: usize = SSIE_FILTER | STIE_FILTER | SEIE_FILTER | LCOFIE_FILTER;
 
-    // Mstatus fields constants
+    /// Constant to filter out writable bits of mip.
```

Comment:
Seems there is a typo ^^

```suggestion
    /// Constant to filter out writable bits of mie.
```